fastlane/lib/fastlane/actions/get_version_number.rb

```diff
@@ -85,9 +104,9 @@ def self.available_options
                                UI.user_error!("Please pass the path to the project, not the workspace") if value.end_with? ".xcworkspace"
                                UI.user_error!("Could not find Xcode project at path '#{File.expand_path(value)}'") if !File.exist?(value) and !Helper.is_test?
                              end),
-          FastlaneCore::ConfigItem.new(key: :scheme,
-                             env_name: "FL_VERSION_NUMBER_SCHEME",
-                             description: "Specify a specific scheme if you have multiple per project, optional",
+          FastlaneCore::ConfigItem.new(key: :target,
```

Comment:
@rs17b check out the [`deprecated` param](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/configuration/config_item.rb#L36) 👍